conf/cluster.yaml.sample

```diff
@@ -39,7 +39,7 @@ cluster:
         - fsm
       - name: s3
         services:
-        - bg_producer
+        - s3bgschedular
```

Comment:
Why using the service name with prefix "s3" when it is already marked under component "s3"
Consider using name such as "bg_scheduler" or "bgscheduler"
Guess the name "schedular" is incorrect...

Reply:
May be bghandler is one possible name.

Reply:
lets call bghandler

Reply:
Fix typo, it should be `bgscheduler`